modules/database/funcs.go

```diff
@@ -229,3 +231,34 @@ func (d *Database) getDBWithName(
 
 	return nil
 }
+
+//
+// Returns a *Database object with specified name and namespace
+//
+func GetDatabaseByName(
+	ctx context.Context,
+	h *helper.Helper,
+	name string,
```

Comment:
Let's call this `databaseName` to make it symmetric with `NewDatabase`

Reply:
ok